lib/ansible/module_utils/distro/_distro.py

```diff
@@ -41,6 +41,39 @@
 
 __version__ = "1.6.0"
 
+# Use `if False` to avoid an ImportError on Python 2. After dropping Python 2
+# support, can use typing.TYPE_CHECKING instead. See:
+# https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING
+if False:  # pragma: nocover
```

Comment:
Consider using the idiom I used in the depresolver code: https://github.com/ansible/ansible/blob/aa4f23d/lib/ansible/galaxy/dependency_resolution/dataclasses.py#L16-L21.

We could even put that try/except into some compat module that could be reused.

Reply:
This code comes from the original `distro` implementation, so it shouldn't be changed unless it's broken.

Reply:
In that case, maybe submit this separately to avoid confusion?